build/test.ps1

```diff
@@ -43,8 +43,8 @@ function Test-One {
 Test-One '../Simulation.sln'
 
 if ($Env:ENABLE_QIRRUNTIME -ne "false") {
-    $qirRuntime = (Join-Path $PSScriptRoot "../src/Qir/Runtime")
-    & "$qirRuntime/test-qir-runtime.ps1"
+    $qirstdlib = (Join-Path $PSScriptRoot "../src/Qir/Runtime")
+    & "$qirstdlib/test-qir-stdlib.ps1"
```

Comment:
(Nit)
```suggestion
    $qirStdlib = (Join-Path $PSScriptRoot "../src/Qir/Runtime")
    & "$qirStdlib/test-qir-stdlib.ps1"
```

Reply:
Updated pattern to sue better variable name and `Join-Path` arguments.